src/test/BiscuitDisplayerTest.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace BiscuitApp\Test;
+use PHPUnit\Framework\TestCase;
+require_once '../vendor/autoload.php';
+
+class BiscuitDisplayerTest extends TestCase
+{
+
+}
```

Comment:
put `namespace` first, then empty line, then `require_once`, then empty line, then `use`

Reply:
in all three files

Reply:
resolved